.github/actions/markdown-lint/action.yml

```diff
@@ -0,0 +1,35 @@
+name: Markdown Linting
+description: >
+  Lints markdown files by wrapping the markdownlint-cli2 action
+  (https://github.com/DavidAnson/markdownlint-cli2-action).
+
+# When updating defaults here also update them in the `markdown-lint` workflow.
+inputs:
+  config:
+    description: Path to a file to use for the base configuration object (defaults to "lombiq.markdownlint.json").
+    default: 'lombiq.markdownlint.json'
+    required: false
+  fix:
+    description: Whether to fix issues automatically for the rules that support it (any truthy value enables).
+    default: ''
+    required: false
```

Comment:
Have you tested this feature? How does this even work? Maybe it would be cool to somehow copy the auto-fixed files into an artifact if that's possible.

Reply:
I haven't, but I'll check!

Reply:
I've implemented this, see https://github.com/Lombiq/GitHub-Actions/actions/runs/9354126597 for the logs and artifact. [The next commit](https://github.com/Lombiq/GitHub-Actions/pull/351/commits/7d0dca2eb9e89a8ba456f92447c5e0213e24d26c) contains those fixes.